topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -236,7 +236,7 @@ Our objective is to find which exon contains the most SNPs. Therefore we have to
 >
 > To find intersection we will be using `intersect intervals` tool from [BEDTools](https://bedtools.readthedocs.io/en/latest/content/overview.html) package.
 >
-> 1. {% tool [bedtools intersect intervals](toolshed.g2.bx.psu.edu/repos/iuc/bedtools/bedtools_intersectbed/2.30.0) %} the intervals of two datasets side-by-side:
+> 1. {% tool [bedtools intersect intervals](toolshed.g2.bx.psu.edu/repos/iuc/bedtools/bedtools_intersectbed/2.30.0+galaxy1) %} the intervals of two datasets side-by-side:
```

Comment:
we like to the keep the workflow as a reflection of the tutorial, could you please update it as well to match these changes?